### ISSUE TITLE
 enable selectedVoice switch

### ISSUE THREAD

**Issue by sallespro** [YerkoPalma/choo-tts#7]
hi, very neat example.
just a reminder to enable `selectedVoice` switch at `onclick()` uncommenting 
`state.tts.selectedVoice = state.tts.voices[document.querySelector('select').value]`

```
  function onclick () {
    // speak with default voice
    state.tts.selectedVoice = state.tts.voices[document.querySelector('select').value]
    emit('tts:speak', document.getElementById('text').value)
  }
```

**Comment by YerkoPalma** [comment]
Hi @sallespro nice catch.
I commented that line to replace it with the [set voice](https://github.com/YerkoPalma/choo-tts#ttsset-voice--ttseventsset_voice) event, but forget to update haha.

**Comment by YerkoPalma** [comment]
fixed in 14fd525
